test/integration/scripts/post-test.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('mz/fs')
+const path = require('path')
+const os = require('os')
+
+const teardown = async () => {
+  const configPath = path.join(os.homedir(), '.contentfulrc.json')
+  const backupPath = path.join(os.homedir(), '.contentfulrc.json.backup')
+  try {
+    const configContent = await fs.readFile(backupPath)
+    console.log('Restoring config file...')
+    await fs.writeFile(configPath, configContent)
+  } catch (e) {
+    if (e.code === 'ENOENT') {
+      // no backup existed, which means no config existed before the test? TODO what to do in this situation
+      console.log('No backup, removing proxies from config...')
+      const configContents = JSON.parse(await fs.readFile(configPath))
+      delete configContents.proxy
+      delete configContents.rawProxy
+      await fs.writeFile(configPath, JSON.stringify(configContents))
```

Comment:
wasn't sure if this was correct thing to do here.

if there is no backup, that means there was no config file before running the tests. should we just remove the file all together? this felt weird in case these scripts are run solo, you could accidentally delete your config.

Reply:
also this assumes config lives at root, which is not necesarily the case.  I'll add a to-do to fix this

Reply:
It would be nice to create a ticket internally also for this